stdlib/runtime/exception.gr

```diff
@@ -29,10 +29,18 @@ export let dangerouslyRegisterBasePrinter = (f) => {
     }
     current = next
   }
+  // Leaks memory (cyclic dep. with Memory) :(
+  //Memory.decRef(WasmI32.fromGrain(dangerouslyRegisterBasePrinter))
+  //Memory.decRef(WasmI32.fromGrain(f))
```

Comment:
Probably let's leave a comment about why we aren't doing the decref here.
```suggestion
  // We don't decRef the closure or arguments here to avoid a cyclic dep. on Memory.
  // This is fine, as this function should only be called once.
```